infra/build/functions/trial_build.py

```diff
@@ -208,6 +208,11 @@ def _do_build_type_builds(args, config, credentials, build_type, projects):
       logging.error('Couldn\'t get project data. Skipping %s.', project_name)
       continue
 
+    # TODO(metzman): Support aarch64 builds.
+    architectures = project_yaml['architectures']
+    if 'aarch64' in architectures:
+      architectures.pop(architectures.index('aarch64'))
```

Comment:
nit for simplicity:
```python
architectures.remove('aarch64')
```

And do we know there is at most one `'aarch64'` in `architectures`?